src/node/psbt.cpp

```diff
@@ -31,6 +32,9 @@ PSBTAnalysis AnalyzePSBT(PartiallySignedTransaction psbtx)
         // Check for a UTXO
         CTxOut utxo;
         if (psbtx.GetInputUTXO(utxo, i)) {
+            if (!MoneyRange(in_amt) || !MoneyRange(utxo.nValue) || !MoneyRange(in_amt + utxo.nValue)) {
```

Comment:
Drop first check, same below.

Reply:
Done, but the check below is necessary. It's how the accumulator is guaranteed to return -1.

Reply:
> but the check below is necessary.

Oh yeah sorry.